docs/user/faq.rst

```diff
@@ -25,3 +25,36 @@ interface. This DNS server must be announced in router advertisements (using
 on *batman-adv*. If your mesh does not have global IPv6 connectivity, you can setup
 your *radvd* not to announce a default route by setting the *default lifetime* to 0;
 in this case, the *radvd* is only used to announce the DNS server.
+
+.. _faq-lost-settings:
+
+Which settings are retained or migrated upon an update?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Gluon provides a :doc:`../dev/web/config-mode`, which allows the configuration via webinterface.
+Under the hood it uses UCI and sets options according to the users input.
+
+There's a recurring misconception about all UCI settings being retained and migrated across
+updates unconditionally. A settings presence does not imply this on its own.
+
+Only if a setting is either configurable via the setupmode, or explicitly referenced
+in this documentation - the wiki or other inofficial places do not count - it is supposed to
+survive the update process.
+
+All other options might disappear or lose functionality in any update.
+
+.. _faq-gluon-preserved:
+
+How can I retain options upon updates anyway?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For settings in the sections *network* and *system* their option *gluon_preserved* can be set to true.
```

Comment:
```suggestion
For settings the sections *network* and *system*, the option *gluon_preserved* can be set to true
in order to preserve the sections. However, this is still at a high risk and may result in broken setups.
```

Reply:
Maybe I'm too tired, but I don't get the drop of the `in`.
Gotta read up on gluon_preserved again :)

Reply:
Alright, debugged it properly and found a mishap. The option is called `gluon_preserve`, not `gluon_preserved`.

Sub-Sections of system and network can be preserved.

@lemoer I think the `in` is correct. Please let me know if this is resolved in your eyes as well.

Reply:
0bd0cf88df037e8fcf2e94fba70b4d434f345c58

Reply:
"nö war keine Absicht"